JiraPS/Public/New-JiraIssue.ps1

```diff
@@ -119,7 +119,27 @@ function New-JiraIssue {
             $name = $_key
             $value = $Fields.$_key
 
-            if ($field = Get-JiraField -Field $name -Credential $Credential -Debug:$false) {
+            $fieldsMatchingName = Get-JiraField -Field $name -Credential $Credential -Debug:$false
```

Comment:
I am not too fond of the way this is coded. Particularly the `[0]`s.

how about this instead?
```pwsh
     if ($field = Get-JiraField -Field $name -Credential $Credential -Debug:$false) {
        if ($field.Count -gt 1) {
            $field = $field | Where-Object { $_.Id -eq $name } | Select-Object -First 1
        }
     }
    if (-not $field) {
        $exception = ([System.ArgumentException]"Invalid value for Parameter")
                        $errorId = 'ParameterValue.InvalidFields'
                        $errorCategory = 'InvalidArgument'
                        $errorTarget = $Fields
                        $errorItem = New-Object -TypeName System.Management.Automation.ErrorRecord $exception, $errorId, $errorCategory, $errorTarget
                        $errorItem.ErrorDetails = "Field name [$name] matches multiple fields by name or by IDs ($($fieldsMatchingName | Out-String)). Please specify field by ID."
                        $PSCmdlet.ThrowTerminatingError($errorItem)
     )
```

POC:
![image](https://github.com/user-attachments/assets/0ade12db-d6e3-4abb-ae89-55fc28d58fce)